src/lib/containers/discussion_forum/DiscussionThread.tsx

```diff
@@ -0,0 +1,171 @@
+import moment from 'moment'
+import React, { useEffect, useState } from 'react'
+import { formatDate } from '../../utils/functions/DateFormatter'
+import {
+  useGetRepliesInfinite,
+  useGetThread,
+} from '../../utils/hooks/SynapseAPI/forum/useForum'
+import { SynapseClient } from '../../utils'
+import { SMALL_USER_CARD } from '../../utils/SynapseConstants'
+import { useSynapseContext } from '../../utils/SynapseContext'
+import {
+  Subscription,
+  SubscriptionObjectType,
+  Topic,
+} from '../../utils/synapseTypes/Subscription'
+import Typography from '../../utils/typography/Typography'
+import { getMessage } from '../DiscussionSearchResult'
+import UserCard from '../UserCard'
+import { getSubscribe } from './ForumTable'
+import { displayToast } from '../ToastMessage'
+import { DiscussionReply } from './DiscussionReply'
+import { Button, FormControl } from 'react-bootstrap'
+import IconSvg from '../IconSvg'
+import Tooltip from '../../utils/tooltip/Tooltip'
+import MarkdownSynapse from '../markdown/MarkdownSynapse'
+import { ObjectType } from '../../utils/synapseTypes'
+
+export type DiscussionThreadProps = {
+  threadId: string
+  limit: number
+}
+
+const FOLLOWING_TEXT = 'You are following this topic. Click to stop following.'
+const UNFOLLOWING_TEXT = 'You are not following this topic. Click to follow.'
+
+export function DiscussionThread(props: DiscussionThreadProps) {
+  const { threadId, limit } = props
+
+  const { accessToken } = useSynapseContext()
+  const [datePosted, setDatePosted] = useState(true)
```

Comment:
Not clear that this is a sort. Can you rename to something like `sortRepliesAscending`?